src/Components/Artwork/Save.tsx

```diff
@@ -45,6 +58,9 @@ export class SaveButton extends React.Component<Props, null> {
           },
         },
       })
+
+      // FIXME: Add non-relay based favorite mechanism
```

Comment:
cool, I loved the example and 'patterns' in the description of the PR (i understand the setup), and was wondering about the mutations (I guess in this case it's _just_ the artwork save). Makes sense!

Reply:
Updated the readme with a TODO note